packages/web-react/src/components/UNSTABLE_Divider/README.md

```diff
@@ -0,0 +1,19 @@
+# UNSTABLE Divider
+
+⚠️ This component is UNSTABLE and there could be breaking changes in the future. Please use it with caution.
```

Comment:
I think this will be linked to this decision in the near future - https://github.com/lmc-eu/spirit-design-system/pull/1454.

I also think that unstable components are some kind of the bleeding edge. So significant change is possible but entirely disappear should not happen, IMHO.